src/site/content/en/blog/origin-trials/index.md

```diff
@@ -55,13 +66,11 @@ trials, as learnings are incorporated and adjustments are made.
    To do so, you are again asked to submit feedback.
    
 {% Aside 'warning' %}
-  Even if an API lands unchanged after a successful origin trial,
-  there is a short period between the end of the origin trial and the date the implementation ships
-  in the browser when the API will not be available.
-  This is by design.
-  If we were to avoid the mandatory total-breakage period,
-  we would bias toward also avoiding breakages in the API surface,
-  and the final shipping API might potentially be worse for it.
+Even if an API lands unchanged after a successful origin trial, there is a short period between the 
+end of the origin trial and the date the implementation ships in the browser when the API will not 
+be available. This is by design. If we were to avoid the mandatory total-breakage period, we would 
+bias toward also avoiding breakages in the API surface, and the final shipping API might potentially 
+be worse for it.
```

Comment:
```suggestion
bias toward also avoiding breakages in the API surface, which are often needed to improve the API.
The final shipping API might be worse for it.
```

Reply:
I'm not in love with this wording, but we should say why API breakages may be needed.

Reply:
Makes sense. 

Also wondering — should maybe avoid the use of 'we'?

Reply:
The injunction agains the word is because it's often ambiguous who 'we' refers to. I don't think there's ambiguity here. Nevertheless, replacing 'we' with 'Chrome' or something equally specific would improve the writing. 

That's a long way of saying, 'yes'.